split.Console/Models/Blackjack/BlackjackCard.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+namespace split.Console.Models.Blackjack
+{
+    public class BlackjackCard
```

Comment:
I think if we go with the interfaces, perhaps, eventually, this class would have to implement ```ICardInterface```, which I think contains only one method ```ShowCard()```, which is a getter to get the ```BlackjackCard``` object.